ldelf/ftrace.c

```diff
@@ -8,19 +8,18 @@
 #include <sys/queue.h>
 #include <types_ext.h>
 #include <util.h>
-#include <user_ta_header.h>
 
 #include "ftrace.h"
 #include "ta_elf.h"
 
 #define MIN_FTRACE_BUF_SIZE	1024
 #define MAX_HEADER_STRLEN	128
 
-static struct __ftrace_info *finfo;
```

Comment:
This and removing the `#include <user_ta_header.h>` is a cleanup.
The cleanup is good, but I would have preferred to have it in a separate commit.

Reply:
> removing the `#include <user_ta_header.h>` is a cleanup.

I need to use `ftrace_buf` in `ftrace.h`, so I moved this header inclusion to `ftrace.h`. Regarding `finfo`, yes its a cleanup to make it a local variable. I thought this was a minimal change and a separate patch would be an overkill. But if you still think otherwise, I will create a separate patch for this.

Reply:
It's not a big deal and I would have let it pass. Do as you like.